src/main/kotlin/de/bund/digitalservice/useid/refresh/RefreshController.kt

```diff
@@ -0,0 +1,38 @@
+package de.bund.digitalservice.useid.refresh
+
+import de.bund.digitalservice.useid.identification.IdentificationSessionService
+import mu.KotlinLogging
+import org.springframework.http.HttpStatus
+import org.springframework.http.ResponseEntity
+import org.springframework.web.bind.annotation.GetMapping
+import org.springframework.web.bind.annotation.RequestMapping
+import org.springframework.web.bind.annotation.RequestParam
+import org.springframework.web.bind.annotation.RestController
+import reactor.core.publisher.Mono
+import java.net.URI
+import java.util.UUID
+
+@RestController
+@RequestMapping("/refresh")
+class RefreshController(private val identificationSessionService: IdentificationSessionService) {
+
+    private val log = KotlinLogging.logger {}
+
+    @GetMapping
+    fun redirectService(@RequestParam sessionId: UUID): Mono<ResponseEntity<Unit>> {
+        return identificationSessionService.findByEIDSessionId(sessionId)
+            .doOnError { exception ->
+                log.error { "error occurred while checking $sessionId;\n ${exception.message}" }
+            }
+            .map {
+                ResponseEntity
+                    .status(HttpStatus.SEE_OTHER)
```

Comment:
303 SEE OTHERS is "Used to redirect after a PUT or a POST, so that refreshing the result page doesn't re-trigger the operation." ([see here](https://developer.mozilla.org/en-US/docs/Web/HTTP/Redirections))
So I would suggest to use 301.

Reply:
After discussion, we keep the 303 SEE OTHERS based on the behaviour of `redirect:` from Spring